EESSI-pilot-install-software.sh

```diff
@@ -326,6 +326,12 @@ fail_msg="Installation of IPython failed, that's unexpected..."
 $EB IPython-7.15.0-foss-2020a-Python-3.8.2.eb -r
 check_exit_code $? "${ok_msg}" "${fail_msg}"
 
+echo ">> Installing WRF 3.9.1.1..."
+ok_msg="WRF installed, it's getting hot in here!"
+fail_msg="Installation of WRF failed, that's unexpected..."
+$EB WRF-3.9.1.1-foss-2020a-dmpar.eb -r
```

Comment:
Two issues popped up while testing the installation of WRF:

* One some systems (depending on how recent the IB network is), I had to set `$OMPI_MCA_pml` to `ucx`; see also https://github.com/EESSI/software-layer/issues/136
* I had to use the customized easyblock from https://github.com/easybuilders/easybuild-easyblocks/pull/2648 to prevent a build failure on `skylake_avx512`

So, this line should be changed to:

```python
OMPI_MCA_pml=ucx $EB WRF-3.9.1.1-foss-2020a-dmpar.eb -r --include-easyblocks-from-pr 2648

Reply:
file updated with final command...